02_configure_host.sh

```diff
@@ -148,8 +150,14 @@ for IMAGE_VAR in $(env | grep "_LOCAL_IMAGE=" | grep -o "^[^=]*") ; do
   if [[ "$IMAGE" =~ "://" ]] ; then
     REPOPATH=~/${IMAGE##*/}
     # Clone to ~ if not there already
-    [ -e "$REPOPATH" ] || git clone "$IMAGE" "$REPOPATH"
-    cd "$REPOPATH" || exit
+    if [ -e "$REPOPATH" ]; then
+       cd "$REPOPATH" || exit
```

Comment:
please one kind of variable name either "$VAR" or $VAR or ${VAR} or "${VAR}". It is different everywhere now. I prefer "${VAR}"